src/main/resources/templates/shopping.html

```diff
@@ -8,8 +8,8 @@
 	<link rel="stylesheet" type="text/css" href="/css/animation.css"/>
 	<link rel="stylesheet" type="text/css" href="/css/slide-toggle.css"/>
 	<link rel="stylesheet" type="text/css" href="/css/shopping.media-queries.css"/>
-	<link rel="shortcut icon" href="images\favicon.ico" type="image/x-icon"/>
-	<link rel="icon" href="images\favicon.ico" type="image/x-icon"/>
```

Comment:
Yeah, on the shoppers page the icon works without the / and, as far as I can tell, it was copied and pasted into the cart that way as well.  But it didn't work in the cart without the /.  However, I just left it alone in the shopper view since it was working to begin with.